### PR TITLE
Fix issues in "extending the identity server" section

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose

Fix toml configs
Fix broken and outdated links

### CODE REVIEW

File: en/docs/learn/configuring-users.md [hunk 1]
```diff
@@ -122,23 +116,24 @@ The user can also be created by calling the
...
+    enable = false
     ```
 
 2.  Open the following Admin Service from SOAP UI:
     [https://localhost:9443/services/RemoteUserStoreManagerService?wsdl  
     ](https://localhost:9443/services/RemoteUserStoreManagerService?wsdl)
 
     !!! info 
-         If you have configured WSO2 IS to use an IP or hostname, replace
+         If you have ocnfiguredWSO2 IS to use an IP or hostname, replace
```
Comment (gomathyk) [inline en/docs/learn/configuring-users.md]: Please check the spelling - configured WSO2

File: en/docs/learn/configuring-users.md [hunk 2]
```diff
@@ -228,16 +223,16 @@ The users also can update their claims by calling the
 `          RemoteUserStoreManager         ` service. If you are new to
 admin services, see [Calling Admin Services](../../develop/calling-admin-services).
 
-1.  Enable the hidden admin service property in the
-    `            <IS_HOME>/repository/conf/deployment.toml          `
+1.  Disable the hidden admin service property in the
+    `            <IS_HOME>/repository/conf/carbon.xml           `
```
Comment (gomathyk) [inline en/docs/learn/configuring-users.md]: Is this going to remain as an xml config?

File: en/docs/learn/configuring-users.md [hunk 3]
```diff
@@ -651,13 +641,13 @@ related to user deletion each time you delete a user:
...
     [event.default_listener.user_deletion]
-    priority= "98"
-    enable = true 
+      priority= "98"
+      enable = false 
     ```
 
-2.  Add the 
+2.  Edit the
     `           <IS_HOME>/repository/conf/identity/identity.xml          `
```
Comment (gomathyk) [inline en/docs/learn/configuring-users.md]: aren't we changing this into toml?

File: en/docs/learn/creating-a-xacml-policy.md [hunk 4]
```diff
@@ -45,80 +45,78 @@ policy using UI according to your preference.
...
+    
+    Ex: ***\[09:00:00+05:00, 16:00:00+05:00\]** time between 09.00am and
+    04.00 pm*
+    
+    Ex: ***(18, 30\]*** *greater than 18 and less than or equal to 30*
     
 
-##### A sample policy example: This policy is defined for accessing “foo” resource
+##### Sample Policy Example 
+A example of a policy is given below. This policy is defined for accessing the “foo” resource.
```
Comment (gomathyk) [inline en/docs/learn/creating-a-xacml-policy.md]: An example right?

File: en/docs/develop/customizing-authentication-error-messages.md [hunk 5]
```diff
@@ -19,14 +19,15 @@ those errors. There are three types of custom errors handled here:
...
+Add the following properties to the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the authenticator to be able to customize error messages.
+
+``` toml
+[authentication.authenticator.basic] 
+name ="BasicAuthenticator"
+enable=true
+[authentication.authenticator.basic.parameters]
+auth_mechanism= "basic"
+UserNameAttributeClaimUri = ""
 ```
```
Comment (ayshsandu) [inline en/docs/develop/customizing-authentication-error-messages.md]: [authentication.authenticator.basic.parameters]
showAuthFailureReason = true

this should be ebough. verify before adding

File: en/docs/learn/creating-a-xacml-policy.md (hunk 4 above)
Comment (sherenewso2) [reply en/docs/learn/creating-a-xacml-policy.md]: fixed with 0b4a567974f56d3b0c866d3f7286017c6ff55bc2

File: en/docs/develop/customizing-authentication-error-messages.md (hunk 5 above)
Comment (sherenewso2) [reply en/docs/develop/customizing-authentication-error-messages.md]: fixed with 0b4a567974f56d3b0c866d3f7286017c6ff55bc2

**Pull request merged**